docs/source/sci_api_tutorial/preparation_remove_solvent.rst

```diff
@@ -0,0 +1,122 @@
+==============================================
+ Preparation: Remove Solvent
+==============================================
+
+Briefs
+==============================================
+
+This science API, named ``enzy_htp.preparation.clean.remove_solvent``,
+removes solvent from the supplied ``enzy_htp.structure.Structure`` class instance 
+(hereafter referred to as ``Structure`` instance).
+
+All ``enzy_htp.structure.Solvent`` class instances (hereafter referred to as
+``Solvent`` instances) will be removed.
+
+Input/Output
+==============================================
+
+**input**: A ``Structure`` instance (no matter it's a protein, polypeptite, or ligand).
+
+.. dropdown:: :fa:`eye,mr-1` How to obtain ``Structure`` instance?
+
+    Structure can be obtained by parsing from a file using one of the StructureParser:
+
+    - `PDBParser <xxx>`_ (Commonly used here)
+    - `PrmtopParser <xxx>`_
+
```

Comment:
Please add a link to the page containing all parsers. refer to QZ's branch. Please unify the format of redirection and only have 1 page for all the information

Reply:
The same for two other docs

Reply:
Done.